### PR TITLE
Fixes a few bugs with tags (filtering, media overlay)

### LINKED ISSUE #679: Edit Media
Show Tags and show Description does not work

### PR DESCRIPTION
**Pull request by wachterjohannes**
__Following bugs fixed:__

* #679 show tags and show description in edit media
* rerender preview when new window is closed (no issue found)
* filtering tags in media edit overlay (part of issue #680)
* filtering tags in tag list (no issue found)

__Tasks:__

- [ ] gather feedback for my changes
- [ ] added changelog line

__Informations:__

| Q             | A
| ------------- | ---
| Tests pass?   | yes
| Fixed tickets | fixes #679 #663 #652
| BC Breaks     | none
| Doc           | none

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ CHANGELOG for Sulu
 * dev-develop
     * ENHANCEMENT #695 [ContentBundle]  Hide textblock sort option when there is only 1 textblock available
     * FEATURE     #634 [AdminBundle]    Created new configuration component, added new configuration for autocomplete
+    * BUGFIX      #681 [TAGBUNDLE]      Fixed filtering of tags in Tag list and Media edit Overlay
```
Comment (danrot) [inline CHANGELOG.md]: Do not introduce new conventions ;-)

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ CHANGELOG for Sulu
...
+    * BUGFIX      #681 [MediaBundle]    Fixed imagick detection
+    * BUGFIX      #681 [TagBundle]      Fixed filtering of tags in Tag list and Media edit Overlay
+    * FEATURE     #634 [All]            Created new configuration component, added new configuration for autocomplete
```
Comment (danrot) [inline CHANGELOG.md]: We really try to avoid All now, I would say the configuration component is part of the AdminBundle

File: CHANGELOG.md (hunk 1 above)
Comment (wachterjohannes) [inline CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (wachterjohannes) [inline CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,10 @@ CHANGELOG for Sulu
 * dev-develop
     * ENHANCEMENT #695 [ContentBundle]  Hide textblock sort option when there is only 1 textblock available
     * FEATURE     #634 [AdminBundle]    Created new configuration component, added new configuration for autocomplete
+    * BUGFIX      #681 [TagBundle]      Fixed filtering of tags in Tag list and Media edit Overlay
+    * BUGFIX      #681 [MediaBundle]    Fixed imagick detection
+    * BUGFIX      #681 [TagBundle]      Fixed filtering of tags in Tag list and Media edit Overlay
```
Comment (danrot) [inline CHANGELOG.md]: Same as two lines before

File: CHANGELOG.md (hunk 2 above)
Comment (dantleech) [inline CHANGELOG.md]: Is it in the Bundle or the Component? If it is in the component then it should not be `Bundle` imo.

**Pull request merged**